src/initialize.F90

```diff
@@ -59,7 +66,7 @@ subroutine openmc_init(intracomm)
 
 #ifdef MPI
     ! Setup MPI
-    call initialize_mpi(intracomm)
+    call initialize_mpi(comm)
```

Comment:
We need to check for a null pointer in case a user called `openmc.lib.init()` without a communicator.